CHANGELOG.md

```diff
@@ -1,363 +1,1031 @@
 # Changelog
 
-All notable changes to the DTS project will be documented in this file.
+All notable changes to this project will be documented in this file.
 
-## v2.0.1-rc3 - 2024-11-21
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-* dts-scripts: Enhance verbose mode + collect cbmem console in hcl report
-* dts-scripts: Add vbt to hcl report
-* dts-scripts: Bump DPP versions for MSI
+## [2.1.0] - 2024-12-09
 
-## v2.0.1-rc2 - 2024-11-04
+### Added
+- recipes-gnome: gcab: add bbapend that disable qa tests
+- recipes-kernel: linux-yocto: add efivars.cfg back to SRC_URI
+- distro: recipes-kernel: linux-yocto: add capsule update configuration
 
-* kernel/linux-yocto: Add CONFIG_SERIAL_8250_DW
-* recipes-extended: sbctl: add RDEPENDS to recipe
-* flashrom: Add TGL chipset detection based on SPI PCI ID
 
-## v2.0.1-rc1 - 2024-10-23
+### Changed
+- conf/distro/dts-distro.conf: ROOT_HOME was missing
+- recipes-bsp: txe-secure-boot: smmstoretool fix buildpaths warning
+- recipes-bsp: coreboot-utils: nvramtool - Fix buildpath warning
+- recipes-connectivity: wolfssl: Ignore [buildpaths] QA ISSUE
+- recipes-gnome: gcab: use insane_skip instead of disabling whole qa check
+- recipes-core: base-files: Mount efivars by default. Needed by FUM
+- distro: recipes-dts: dts-scripts: bump revision
+- Add generate-changed-recipes script + specify PV for git recipes
+- .github/workflows: Deploy manifest to boot.dasharo.com
+- Use git-cliff to generate changelog
+- recipes-bsp/coreboot-utils: Use 4.21 as base version for coreboot-utils packages
+- README: Link Zarhus release process
+- Change git-cliff config
+- .github/workflows: Fix getting DTS_VER in manifest deployment
+- Move generate-changelog.sh to scripts
+- Fill up CHANGELOG after rebase
+- Add extension to generate-changed-recipes
+- README: Explain parameter in generate-changelog.sh
+- .github/workflows: Add workflow with DTS tests
+- .github/workflows: Add build.yml reusable workflow
+- .github/workflows/test.yml: Print test statuses to stdout
+- .github/workflows/test.yml: Specify Python version (3.12)
+- .github/workflows/test.yml: Fix
+- .github/workflows/test.yml: Create hdd.qcow using `touch`
+- workflow: Fix build failure reported as success
+- dts-distro.conf: bump to 2.1.0
+- .github/workflows/test.yml: Fix job name in `needs`
+- .github/workflows/test.yml: Hotfix
+- .github/workflows/test.yml: Add missing backslashes
+- dts-scripts: Improve UI/UX
+- dts-scripts: Hotfix
```

Comment:
Do we need so many commits about workflow in changelog?
And some commits e.g.:

* `dts-scripts: Hotfix`
* `.github/workflows/test.yml: Fix`

don't add anything as you can't deduce from changelog what was fixed/changed.

Reply:
You're right. I removed some notes https://github.com/Dasharo/meta-dts/compare/8a35e7fc1219311ddbad68dbd7927ecda4322a5f..3ed77b24899849f939d8bf1a173027837e63bed4